nuto/mechanics/interpolation/TypeDefs.h

```diff
@@ -7,19 +7,18 @@ namespace NuTo
 
 typedef Eigen::VectorXd NodeValues;
 // typedef Eigen::VectorXd NaturalCoords;
-typedef Eigen::VectorXd ShapeFunctions;
 typedef Eigen::MatrixXd DerivativeShapeFunctionsNatural;
 typedef Eigen::MatrixXd DerivativeShapeFunctionsGlobal;
 typedef Eigen::MatrixXd BMatrixStrain;
 typedef Eigen::MatrixXd BMatrixGradient;
 typedef Eigen::MatrixXd NMatrix;
```

Comment:
Can we not get rid of all of these, except for `NaturalCoords` and `GlobalCoords`?

Reply:
I will remove them.